matching-core/src/test/java/com/graphhopper/matching/util/HmmProbabilitiesTest.java

```diff
@@ -1,18 +0,0 @@
-package com.graphhopper.matching.util;
-
-import org.junit.Test;
-import static org.junit.Assert.*;
-
-/**
- *
- * @author Peter Karich
- */
-public class HmmProbabilitiesTest {
-
-    @Test
-    public void testTransitionLogProbability() {
-        HmmProbabilities instance = new HmmProbabilities();
-        // see #13 for the real world problem
-        assertEquals(0, instance.transitionLogProbability(1, 1, 0), 0.001);
```

Comment:
Because I needed to remove the timeDiff check (see [above](https://github.com/graphhopper/map-matching/pull/88#discussion_r92573679)), which made this unit test obsolete.